app/src/main/java/com/graymandev/mvvmfoundation/view/intro/IntroActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.graymandev.mvvmfoundation.view.intro
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.graymandev.mvvmfoundation.databinding.ActivityIntroBinding
+import com.graymandev.mvvmfoundation.databinding.ActivitySplashBinding
+
+class IntroActivity : AppCompatActivity() {
```

Comment:
Please use BindingActivity rather than AppCompatActivity. Take MainActiviy as reference

Reply:
My excuses bebo, I just focused in the splash screen activity. I will updated it soon ;).